src/haddock/modules/analysis/caprieval/defaults.cfg

```diff
@@ -3,9 +3,10 @@ irmsd = true
 fnat = true
 lrmsd = true
 ilrmsd = true
-irmsd_cutoff = 10.0
+irmsd_cutoff = 5.0
 fnat_cutoff = 5.0
 lrmsd_cutoff = 10.0
 receptor_chain = 'A'
```

Comment:
Now we are working on taking the segIDs directly from the input PDBs. What if they don't use A and B?  Does it mean the user would have to define those for things to work?

Reply:
Something we need to refactor across all modules once we implement it.

Reply:
But currently the code takes the segID from the input PDB and does not overwrite it, correct.
So we have to assume (for the time being) that the molecules have alphabetical segIDs

Reply:
Yep this is why for now the receptor and ligand chains are passed as a parameter, once we implement it we need to refactor it to get from the internal structure instead.